generators/Options.fs

```diff
@@ -49,16 +68,22 @@ let private normalizeStatus status =
     | Some "missingdata"   -> Some MissingData
     | Some "custom"        -> Some Custom
     | Some "deprecated"    -> Some Deprecated
-    | Some "all"           -> None
+    | Some "outdated"      -> Some Outdated
+    | Some "all"           -> Some All
     | Some _               -> failwith "Invalid status" 
-    | None                 -> Some Implemented
+    | None                 -> None
 
 let private mapOptions (options: CommandLineOptions) =
     { Exercise = normalizeExercise options.Exercise
       Status = normalizeStatus options.Status
       CanonicalDataDirectory = normalizeCanonicalDataDirectory options.CanonicalDataDirectory
       CacheCanonicalData = options.CacheCanonicalData }
 
+let conflictingStatusAndExerciseParams (parsed:Parsed<CommandLineOptions>) =
+    let s = parsed.Value.Status |> String.IsNullOrWhiteSpace |> not
```

Comment:
Could you give this a more descriptive name? E.g. `statusParameterProvided` or `statusParameterSpecified`?